FRENCH/src/ch05-03-method-syntax.md

```diff
@@ -16,12 +16,12 @@ instance of the struct the method is being called on.
 
 Les *méthodes* sont similaires aux fonctions : on les déclare avec le mot-clé
 `fn` et leur nom, elles peuvent avoir des paramètres et une valeur de retour, et
-elles contiennent du code qui est exécuté quand on les appelle depuis un autre
-endroit. Cependant, les méthodes diffèrent des fonctions en ce qu'elles sont
+elles contiennent du code qui est exécuté quand on les utilise depuis un autre
```

Comment:
Why use the imprecise verb `utiliser` instead of the more technical, well-suited verb `appeler`?

Reply:
I've got some doubts on the conjugation of this verb, so I've changed the word, but I now realise it's the same problem, sorry :sweat:
So I will restore your word, `appelle`, very soon but I'm still not sure about the conjugation ... :confused: